heliopy/data/mms.py

```diff
@@ -7,131 +7,1773 @@
+            (local, remote), where `local` and `remote` are lists.
+        '''
+
+        # Search locally if offline
+        if self.offline:
+            local_files = self.local_file_names()
+            remote_files = []
+
+        # Search remote first
+        #   - SDC is definitive source of files
+        #   - Returns most recent version
+        else:
+            remote_files = self.file_names()
+
+            # Search for the equivalent local file names
+            local_files = self.remote2localnames(remote_files)
+            idx = [i for i, local in enumerate(local_files)
+                   if os.path.isfile(local)
+                   ]
+
+            # Filter based on location
+            local_files = [local_files[i] for i in idx]
+            remote_files = [remote_files[i] for i in range(len(remote_files))
+                            if i not in idx
+                            ]
+
+        # Filter based on time interval
+        if len(local_files) > 0:
+            local_files = filter_time(local_files,
+                                      self.start_date,
+                                      self.end_date
+                                      )
+        if len(remote_files) > 0:
+            remote_files = filter_time(remote_files,
+                                       self.start_date,
+                                       self.end_date
+                                       )
+
+        return (local_files, remote_files)
+
+    def version_info(self):
+        '''
+        Obtain version information from the SDC.
+
+        Returns
+        -------
+        vinfo : dict
+            Version information regarding the requested files
+        '''
+        self._info_type = 'version_info'
+        response = self.post()
+        return response.json()
+
+
+def _datetime_to_list(datetime):
+    return [datetime.year, datetime.month, datetime.day,
+            datetime.hour, datetime.minute, datetime.second,
+            datetime.microsecond // 1000, datetime.microsecond % 1000, 0
+            ]
+
+
+def burst_data_segments(start_date, end_date, team=False):
+    """
+    Get information about burst data segments.
 
+    Parameters
+    ----------
+    start_date : `datetime`
+        Start date of time interval for which information is desired.
+    end_date : `datetime`
+        End date of time interval for which information is desired.
+    team : bool=False
+        If set, information will be taken from the team site
+        (login required). Otherwise, it is take from the public site.
 
-def _validate_data_rate(data_rate):
-    allowed_rates = ['slow', 'fast', 'brst', 'srvy', '']
-    if data_rate not in allowed_rates:
-        raise ValueError(
-            'Data rate {} not in list of allowed data rates: {}'.format(
-                data_rate, allowed_rates))
+    Returns
+    -------
+    data : dict
+        Dictionary of information about burst data segments
+            DATASEGMENTID
```

Comment:
This should either be a proper list, or a table (see http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#tables)